ui/app/lib/console-helpers.ts

```diff
@@ -162,7 +211,11 @@ export function logFromError(error, vaultPath, method) {
   return { type: 'error', content };
 }
 
-export function shiftCommandIndex(keyCode, history, index) {
+interface CommandLog {
+  type: string;
+  content?: string;
+}
```

Comment:
Great idea to put the interface type definitions before each function :) I think it makes it more intuitive to know what types are needed for the specific function.